sbin/cubeit

```diff
@@ -500,6 +521,7 @@ IFS=$OLDIFS
 	    # create a configuration that can be read by the cubeit-installer
 	    if [ -v TARGET_CONFIG_FILE -a -n "$TARGET_CONFIG_FILE" ]; then
 		cat $TARGET_CONFIG_FILE |
+			sed "s|ARTIFACTS_DIR=.*|ARTIFACTS_DIR=${INSTALLER_TARGET_IMAGES_DIR}|" | \
```

Comment:
The previous modification was just for HDINSTALL_CONTAINERS variable because they are places under the new "$ARTIFACTS_DIR/containers" subfolder. Somehow the kernel/essential/ramfs rootfs are still placed under the $ARTIFACTS_DIR, so we need to make sure that this is correctly set too.